integration/raft/config_test.go

```diff
@@ -1077,6 +1077,75 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
 		})
 
+		When("an evicted node is added back while it's offline", func() {
+			It("can start with correct new raft id", func() {
+				o1 := network.Orderer("orderer1")
+				o2 := network.Orderer("orderer2")
+				o3 := network.Orderer("orderer3")
+
+				orderers := []*nwo.Orderer{o1, o2, o3}
+
+				By("Waiting for them to elect a leader")
+				findLeader(ordererRunners)
+
+				By("Creating a channel testchannel")
+				network.CreateChannel("testchannel", o1, peer)
+
+				assertBlockReception(map[string]int{
+					"testchannel":   0,
+					"systemchannel": 1,
+				}, []*nwo.Orderer{o1, o2, o3}, peer, network)
+
+				By("Killing the first orderer")
+				ordererProcesses[0].Signal(syscall.SIGTERM)
+				Eventually(ordererProcesses[0].Wait(), network.EventuallyTimeout).Should(Receive())
+
+				// We need to wait for stabilization, as we might have killed the leader OSN.
+				By("Waiting for the channel to stabilize after killing the orderer")
+				assertBlockReception(map[string]int{
+					"testchannel":   0,
+					"systemchannel": 1,
+				}, []*nwo.Orderer{o2, o3}, peer, network)
+
+				By("Removing the first orderer from the application channel")
+				extendNetwork(network)
```

Comment:
why do you need to call **extendNetwork** and **refreshOrdererPEMs** ? I thought we can just remove and re-add an existing orderer?

Reply:
i think this was adhering to the pattern [here](https://github.com/hyperledger/fabric/blob/251e989da1a7fe9eb7b4507ba093b140a93660a6/integration/raft/config_test.go#L1110-L1111) to pick up node cert, which could be read directly instead.